### PR TITLE
Run derivatives if they're missing from Plum

### LINKED ISSUE #263: PlumImporter should generate derivatives if it can't find a derivative file

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #263

### CODE REVIEW

**Comment by tpendragon** [comment]
Given that some derivatives legitimately can't generate with this PR, I also added some commits to
1. Let the manifest still render in UV
2. Make the catalog page not break if it's set to a thumbnail without a derivative.

File: app/services/plum_importer.rb [hunk 1]
```diff
@@ -31,6 +31,12 @@ def generate_derivative(member)
       handlers: change_set_persister.handlers
     )
     derivative_change_set_persister.save(change_set: file_change_set)
+  rescue Errno::ENOENT
```
Comment (hackmastera) [inline app/services/plum_importer.rb]: It's a little confusing that this method is called generate_derivative but I think from conversation that it actually imports / saves the derivative. It looks like it is saving from a file location; are these applications running on the same box?

**Comment by hackmastera** [comment]
If there are derivatives that still aren't generating, should those be logged somewhere for future attention?

File: app/services/plum_importer.rb (hunk 1 above)
Comment (tpendragon) [reply app/services/plum_importer.rb]: :+1: I'll fix the name.

**Comment by tpendragon** [comment]
@hackmastera Take another look?

**Pull request merged**